CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## v4.4.0 (not published yet)
+## v4.4.0 (May 28, 2021)
```

Comment:
Do we need to update the readme compatibility table for the theme?

Reply:
I'm out of the loop on that table... @emclaug2 ??

Reply:
Yeah I think this entry would be appropriate:


| `^4.4.0`                      | `^6.1.0`                    | `^10.x`                       |